documentation/element-api/client-server-rpc.asciidoc

```diff
@@ -32,61 +33,48 @@ public void setExpanded(Component component) {
 
-Notice that `Element.executeJavaScript` method was introduced in Vaadin 12. For previous Vaadin Platform versions it is necessary to access the `Page`.
+*Example*: Using the `executeJavaScript` method.
 
 [source, java]
 ----
 public void complete() {
-    Page page = UI.getCurrent().getPage();
-    page.executeJavaScript("$0.complete()", this);
+    getElement().executeJavaScript("$0.complete()", this);
 }
 ----
 
-[NOTE]
-Please consult and <<../creating-components/tutorial-component-basic#, Creating a Simple Component Using the Element API>> for creating a new component using element API.
+It is also possible to call the `executeJavaScript` method to access methods and fields of a Web Component.
 
-== Call a server-side method from the client side
 
-A method defined in the server side can be called from the client side in 2 different ways:
+== Calling Server-side Methods from the Client
 
-. Using the `@EventHandler` annotation to register a server-side method as an event handler.
-. Using the `@ClientCallable` annotation to invoke a server-side method from the client side.
+You can call a server-side method from the client side using either the `@EventHandler` or `@ClientCallable` annotation.
 
-=== Event handler method
+=== @EventHandler Annotation
 
-The `@EventHandler` annotation publishes the annotated method so it can be invoked from the client side as template event handlers.
-For more information about `@EventHandler`, please consult <<../polymer-templates/tutorial-template-event-handlers",Handling User Events in a PolymerTemplate>>.
+The `@EventHandler` annotation allows you to register a server-side method as an event handler. It publishes the annotated method and allows it to be invoked from the client side as a template event handler. See <<../polymer-templates/tutorial-template-event-handlers#,Handling User Events in a PolymerTemplate>> for more.
 
-=== Client delegate method
+=== @ClientCallable annotation
 
-Another way to call a server side method is `@ClientCallable` annotation which marks a template method as the
-method which should be called from the client side code using notation `this.$server.serverMethodName(args)`.
+The `@ClientCallable` annotation allows you to invoke a server-side method from the client side. It marks a template method as a method that can be called from the client side using the `this.$server.serverMethodName(args)` notation.
 
-It can be used anywhere in your client-side Polymer class implementation. You can pass your own arguments in this method.
-Just make sure that their types match to method declaration on the server side.
+You can use it anywhere in your client-side Polymer class implementation, and can pass your own arguments in the method. Note that the types should match the method declaration on the server side. 
```

Comment:
```suggestion
You can call the method from your client-side code and can pass arguments to the method. Note that the types should match the method declaration on the server side. 
```

Let's remove the Polymer specific part because this is usable in other ways too. I'll create an issue for missing documentation that has been promised a long time ago, but let's not fix that today.